CONTRIBUTORS.rst

```diff
@@ -358,6 +359,7 @@ Contributors
 * Wesley van Lee
 * Md Arifin Ibne Matin
 * Tom Usher
+* Kalob Taulien
```

Comment:
Core team members should just go on the first list. Surprised you weren't on here already... :-)